cluster-autoscaler/loop/trigger.go

```diff
@@ -43,19 +43,25 @@ type scalingTimesGetter interface {
 	LastScaleDownDeleteTime() time.Time
 }
 
+type provisioningRequestProcessTimeGetter interface {
```

Comment:
nit: please make naming format more consistent, e.g. existing "scaling..." vs "Process" in the struct name vs "Processed" in the method name, existing "TimesGetter" vs your "TimeGetter".

Reply:
Converted Struct name to plural present continuous and method name to simple present.